onvif/client.py

```diff
@@ -289,6 +293,37 @@ async def close(self):
         for service in self.services.values():
             await service.close()
 
+    def get_snapshot(self, url, basic_auth):
```

Comment:
Since we are defaulting to digest anyway, let's change this to have a `False` default for the basic_auth param.
```suggestion
    def get_snapshot(self, url, basic_auth=False):
```